BT-Tracking/AppDelegate.swift

```diff
@@ -81,6 +81,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         generalSetup()
         setupInterface()
         setupBackgroundMode(for: application)
+        clearKeychainIfNeeded()
```

Comment:
Tohle by melo byt asi jako prvni vec, urcite pred setupInterface, kde uz to bude cist ty klice

Reply:
ok, upraveno. zamerguju, protoze potrebuju testovat